NSS-Day-Cohort-42/rare-fakes-news
# Cw tags2- Admins can edit Tags

#116, #115 -Can edit tags if admin
​
## Description
Can edit tags in tag management if user is an admin
​
## How do I access this branch?
1.git fetch --all
2. git checkout branch cw-tags2
​
## Motivation and Context
Adding edit tag to webpage
​
## How Has This Been Tested?
1.Make sure you have both pull requests serverside and clientside
2. Login as admin go to tag management
3. Hit edit tag button and edit tag.
4. Make sure its changed in database
​
## Screenshots (if appropriate):
​
## Types of changes

- [x ] New feature (non-breaking change which adds functionality)
​
## Checklist:


- [x ] My code follows the code style of this project.
- [x ] My change requires a change to the documentation.
- [x ] I have updated the documentation accordingly.
